libcodechecker/analyze/analysis_manager.py

```diff
@@ -364,15 +371,41 @@ def check(check_data):
                         LOG.debug(str(ex))
                         other_files = set()
 
+                    mentioned_files_dependent_files = set()
+                    for of in other_files:
+                        mentioned_file = os.path.abspath(of)
```

Comment:
This should be made absolute with respect to the working directory of the build action and not the working directory of the CodeChecker script.

Reply:
In case of clangsa we get abs paths from Clang, but in case of clang-tidy it may be relative. So yep I am changing it.